runtime/lua/vim/lsp.lua

```diff
@@ -613,6 +613,8 @@ do
     if tbl_isempty(all_buffer_active_clients[bufnr] or {}) then
       return
     end
+
+    util.buf_version_set(bufnr, changedtick)
```

Comment:
just put `all_buffer_versions` somewhere it can be accessed directly and access it directly. One line getters/setters does not add anything of value, there is no "interface boundary" between different lsp submodules.

Reply:
Alright, I exposed it as `util.buf_versions`